tools/coverage/docs/local_presubmit.md

```diff
@@ -0,0 +1,11 @@
+# Local Pre-submit Diff
+
+A pre-submit check, to see how the coverage of your local repository compare to the latest successful postsubmit run, can be run locally in the following way
+
+## Steps
+
+1. `go get k8s.io/test-infra/robots/coverage`
```

Comment:
You're downloading an external tool to perform the local coverage diff. Is it intended? If so, what's the connection between that tool and the coverage tool? How do we ensure they produce the same results?

Reply:
Yes. It is intended. This script is not to test the changes made to the tool here, but to see the coverage difference between a user's local branch and a upstream base branch. The external tool was derived from this tool but may diverge as it was ported to Kubernetes repo. By design, both of them should produce the same result, but that cannot be ensured unless we converge them.

Reply:
This is important and can possibly save someone from a long headache trying to understand differences in coverage. Please add this explanation as a note to the doc.